README.md

```diff
@@ -174,20 +174,19 @@ You can either configure it in your `tsconfig.json` or in your `rollup.config.js
 
 ```js
 // Vue JSX
-import vueJsx from 'rollup-plugin-vue-jsx-compat'
 import { swc, defineRollupSwcOption } from 'rollup-plugin-swc3';
 
 export default {
   input: 'xxxx',
   output: {},
   plugins: [
-    vueJsx(),
     swc(defineRollupSwcOption({
       jsc: {
-        transform: {
-          react: {
-              pragma: 'vueJsxCompat'
-          }
+        experimental: {
+          plugins: [['swc-plugin-vue-jsx', {}]]
```

Comment:
```suggestion
          plugins: [['swc-plugin-vue-jsx', {}]] // npm i swc-plugin-vue-jsx
```

Mention that the swc plugin should be installed first.

Reply:
> Mention that the swc plugin should be installed first.

Yeah. I'm add it.